packages/terra-functional-testing/tests/wdio/terra-validates-spec.js

```diff
@@ -48,6 +48,18 @@ Terra.describeViewports('Terra.validates', ['small', 'large'], () => {
 
       expect(caughtError.message).toEqual('[terra-functional-testing:screenshot] Terra.validate.screenshot requires a unique test name as the first argument.');
     });
+
+    it('should fail with invalid selector', () => {
+      let caughtError;
+
+      try {
+        Terra.validates.screenshot('invalid selector', { selector: 'invalid-selector' });
+      } catch (error) {
+        caughtError = error;
+      }
+
+      expect(caughtError.message).toEqual('[wdio-visual-regression-service:makeDocumentScreenshot] Failed to capture the element using the "invalid-selector" selector. Either update the test document to include this selector or use a different selector that exists on the document.');
```

Comment:
jest has a toThrow method to capture error messages - could that be used here instead of a try/catch? try/catch is a bit clunky

Reply:
.toThrow is certainly cleaner and easier: https://github.com/cerner/terra-toolkit/pull/605/commits/ad8e15eb67ba4838ff642dde18e0590e207c2386